packages/typegpu/src/tgsl/generationHelpers.ts

```diff
@@ -517,18 +518,36 @@ function applyActionToSnippet(
   }
 }
 
-export function convertToCommonType(
-  ctx: GenerationCtx,
-  values: Snippet[],
-  restrictTo?: AnyData[],
+export type ConvertToCommonTypeInfo = {
```

Comment:
Tiny nitpick, we commonly use the `Options` suffix for the named function parameters type
```suggestion
export type ConvertToCommonTypeOptions = {
```